source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/lib/index.ts

```diff
@@ -0,0 +1,133 @@
+/**
```

Comment:
I can't comment on the digital file, but we need to update the architecture diagram to be EventBridge